### ISSUE TITLE
The Grey Box Around the Form when Form Loses Focus

### ISSUE THREAD

**Issue by mobedi** [viperneo/winforms-modernui#322]
I have obtained the latest commit of the master code. Is there anyone else who sees the thick grey frame around the form when it loses focus? Is there a way to avoid it or at least refresh the form when it gains focus?
![capture](https://cloud.githubusercontent.com/assets/2116627/9351077/85a54bc4-4656-11e5-9f7d-774e6c4d0c58.PNG)
N.B. The box becomes more grey when I click on the form. It is supposed to go away when the form has gained the focus back right? For some reason, it stays this way.

**Comment by DBenS** [comment]
Yes, I had the same problem and I had put a message with no response in #303.

I had "partially" solved my problem through a "Sub", which perform:

        Me.Activate()
        Me.ResizeRedraw() = True
        Me.ResumeLayout()
        Me.Focus()

It´s **not** a fix, of course, but can solve the problem. The sub is called everytime the form get focus.

**Comment by mobedi** [comment]
@DBenS Which event should we put this code in?

**Comment by DBenS** [comment]
1) After call any form; and/or
2) In the GotFocus() event of form

Obviously it cannot avoid the border if the user changes the focus to another application - and because of this I said it´s not a solution but a partial one...

I guess the problem can be the lack of a Redraw() in the LostFocus() event in the C# main project - but I´m really bad in C#, enough to avoid the necessary investigation in the Metroframework.

**Comment by mobedi** [comment]
@DBenS I have put the code in the form's `Activated` event but my form is still growing. Maybe VB does something C# does not? 
private void Form1_Activated(object sender, EventArgs e)
{
    this.ResizeRedraw = false;
    this.ResumeLayout();                
}

**Comment by mobedi** [comment]
@DBenS I tried adding a C# version of your code to my form's `GotFocus` event but I see no changes unfortunately. But I guess I can live with this for a little while. May I ask if you also have the https://github.com/viperneo/winforms-modernui/issues/323 problem as well? Because this one is really strange.

**Comment by DBenS** [comment]
WOW!  Yes, I´m having the same problem and I hadn´t viewed it before! I will interact at the #323.

**Comment by DBenS** [comment]
@mobedi , the solution of #323 really caused a benefit in that case. What do you think about close it? Hugs!
